lib/dnsmadeeasy-rest-api.rb

```diff
@@ -59,15 +59,15 @@ def records_for(domain_name)
   end
 
   def find(domain_name, name, type)
-    records = records_for(domain_name)
-    records['data'].detect { |r| r['name'] == name && r['type'] == type }
+    records = get "/dns/managed/#{get_id_by_domain(domain_name)}/records?recordName=#{name}&type=#{type}"
```

Comment:
We don't have `#find` implemented in [`nbuild`](https://github.com/3dna/nbuild/blob/master/lib/nation_builder/dns/dns_made_easy_adapter.rb), would that be a separate add after this?